.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { config } = require('@dhis2/cli-style')
+
+module.exports = {
+    extends: [config.eslintReact, 'plugin:@typescript-eslint/recommended'],
+    settings: {
+        'import/resolver': {
+            node: {
+                extensions: ['.js', '.jsx', '.ts', '.tsx'],
+            },
+        },
+    },
+    rules: {
+        'import/extensions': 'off',
```

Comment:
I think we decided to have these (not sure why, though)

Reply:
Import extensions is not compatible with typescript as far as I know. You would also need to enable `allowImportingTsExtensions`.